_data/errors.yml

```diff
@@ -118,4 +118,78 @@ oidc:
         We are no longer supporting the id_token_hint attribute as a method of logging out.
         ##### What to do:
         - Update your logout request to utilize the client_id attribute.
-        More information about the logout request parameters can be found here: [https://developers.login.gov/oidc/#logout-request](https://developers.login.gov/oidc/#logout-request){:target="_blank"}.
+        More information about the logout request parameters can be found here: [https://developers.login.gov/oidc/logout/#logout-request](https://developers.login.gov/oidc/logout/#logout-request){:target="_blank"}.
+  token:
+    - title: is expired
+      id: token-expired
+      accordion-id: token_expired
+      content: |
+        ##### Why it's happening
+        In the authentication response, a `code` value is returned that is then used in the `token` endpoint request. The code is intended to be used immediately (within 15 minutes) after it has been generated. This error may mean that the user paused for too long during the authentication attempt.
+        ##### What to do:
+        - Attempt another authentication attempt without pauses.
```

Comment:
Suggest changing to "Try another authentication attempt without pausing."

Reply:
updated in https://github.com/18F/identity-dev-docs/pull/393/commits/cf8850248cc9e78c13667ec2062a5d7e12dd8aa8